### PR TITLE
materials-designer feature/SOF-5214

### PR DESCRIPTION
**Pull request by adewyer**
Change lattice update from from scalar to additive

### CODE REVIEW

**Comment by adewyer** [comment]
Moved value to made-js, so we don't need to set in materials-designer.

File: src/reducers/Material.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ function materialsUpdateOne(state, action) {
     const index = action.index || state.index;  // not passing index when modifying currently displayed material
     const material = action.material.clone();   // clone material to assert props re-render
     material.isUpdated = true;                  // to be used inside components
+    material.isNonPeriodic = action.material.isNonPeriodic;
```
Comment (adewyer) [inline src/reducers/Material.js:22]: Without this line the state does not retain the `isNonPeriodic` value

Screen Shot 2021-11-29 at 7 50 18 PM


With the line the value is retained

Screen Shot 2021-11-29 at 7 56 01 PM

Comment (timurbazhirov) [reply src/reducers/Material.js:22]: Then we should look into the `clone()` function (or Material class itself and its `toJSON`) and figure out why it doesn't retain the value

Comment (adewyer) [reply src/reducers/Material.js:22]: I will do that.

**Comment by adewyer** [comment]
No changes to materials-designer were needed for updating the lattice at this time. Lattice updates to come for materials-designer in SOF-5215.

**Pull request closed without merging**